### PR TITLE
Add top slider

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Added top slider 

Fixes
- Change the heading of the card to normal text.

Preview: https://deploy-preview-11--autodecore.netlify.app/

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -67,6 +67,17 @@
   height: 200px;
```
Comment (Shivanshsharma01) [inline css/home.css:67]: can use rem here

**Review by Shivanshsharma01 — COMMENTED** [review]
apart from one change suggested, the Code is clean and well structured, good use of variables

**Review by prachi-sahani — APPROVED** [review]
Well structured code! Seems good to merge 👍🏻

File: css/home.css (hunk 1 above)
Comment (junaidshaikh-js) [reply css/home.css:67]: Yes, but I want the figures to be specifically 200px and not dependent on the root font size.

**Review by hinanshisuthar — APPROVED** [review]
Good use of variables. Clean code. Good practices followed. Ready to merge!

**Pull request merged**